ui/bottompanel.go

```diff
@@ -46,6 +54,9 @@ func NewBottomPanel(p *player.Player, contr *controller.Controller) *BottomPanel
 	p.OnStopped(func() {
 		bp.Controls.SetPlaying(false)
 	})
+	p.OnLoopModeChanged(func(mode string) {
```

Comment:
I would route this callback through the PlaybackManager as well. The eventual goal is that only the playbackmanager has a direct dependency on the Player (allowing for pluggable audio backend implementations like DLNA casting). The code isn't there yet but as much as possible I'd like *new* code to conform to that, to make the eventual refactor easier.